qmf/src/libraries/qmfclient/qmailmessage.cpp

```diff
@@ -4169,7 +4179,11 @@ QByteArray QMailMessagePartContainerPrivate::boundary() const
 
 void QMailMessagePartContainerPrivate::setBoundary(const QByteArray& text)
 {
-    _boundary = text;
+    if (text == QByteArray("NIL")) {
```

Comment:
If we don't get a valid boundary here, will we see the actual value later when the message data is received?